src/jit/codegencommon.cpp

```diff
@@ -7595,20 +7598,22 @@ void CodeGen::genProfilingLeaveCallback(unsigned helper /*= CORINFO_HELP_PROF_FC
         if (RBM_ARG_0 & gcInfo.gcRegGCrefSetCur)
         {
             attr = EA_GCREF;
-            gcInfo.gcMarkRegSetGCref(RBM_PROFILER_RET_SCRATCH);
+            gcInfo.gcMarkRegSetGCref(RBM_PROFILER_RET_SCRATCH1 | RBM_PROFILER_RET_SCRATCH2);
```

Comment:
On Arm32, a gc ref/byref would be 4-bytes in size. If ARG_0 is a gc ref/byref, why are we marking RET_SCRATCH2  as ref/byref in gcinfo?

Note that this comment also applies to line 7606